### PR TITLE
endpoint checks

### PR DESCRIPTION
**Pull request by NoelV94**

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
...
 
 module.exports = {
-  API_KEY: process.env.API_KEY,
-  API_URL: process.env.API_URL || 'https://verify.vouched.id/api'
+  API_ADMIN_KEY: process.env.API_ADMIN_KEY,
+  API_PRIVATE_KEY: process.env.API_PRIVATE_KEY,
+  API_PUBLIC_KEY: process.env.API_PUBLIC_KEY,
+  API_URL: process.env.API_URL || 'https://verify.stage.vouched.id/api',
+  API_GRAPHQL_URL: process.env.API_GRAPHQL_URL || 'https://verify.stage.vouched.id/graphql',
+  ONBOARD_URL: process.env.ONBOARD_URL || 'https://onboard.prod.vouched.id/api',
```
Comment (natarajaya) [inline src/config.js:9]: Should the default value be `https://onboard.stage.vouched.id/api`?

Comment (NoelV94) [reply src/config.js:9]: added change to 2895 @natarajaya

**Comment by natarajaya** [comment]
LGTM, but would prefer so @NickLamp or @marcusoliver to have a look at this as well.

**Review by marcusoliver — COMMENTED** [review]
the tests here look good. 
however, all the tests are for negative cases (not authn/authz). do we need test(s) for the positive cases too?

**Comment by NoelV94** [comment]
> the tests here look good.
> however, all the tests are for negative cases (not authn/authz). do we need test(s) for the positive cases too?

@marcusoliver Positive graphql cases require an auth token. Cannot add that. Also, these tests will be taken out in a few weeks once we remove the graphql endpoints

**Review by NickLamp — APPROVED** [review]

**Pull request merged**